### DISCUSSION TITLE
Install Whisper on Google Colaborate

### DISCUSSION THREAD

**Discussion by AllHellkiller** [openai/whisper#1230] [General]
I am not very code savy.I tried to follow every Youtube and article I could find to run whisper on Goog;le Colaborate.Can some one please walk me through the install or provide an alternate way to run Whisper off of my laptop. My laptop is old and cant run if a bear was chasing it .I cant afford anything else>thanks

**Comment by EtienneAb3d** [comment]
Here is an example you can clone and adapt:
https://colab.research.google.com/drive/1-GpXaNaGFXKX9VXl60JGVVrGO41t09KA?usp=sharing

**Reply by perplexed-paperclip** [reply]
Hello - I'm not the original poster, but as another coding novice, this does look quite useful!
If it's okay, may I ask for step-by-step instructions on how to correctly operate and run files through this version of Whisper?
Also, is there a way for the end output to have timestamps? The tutorial I was using before (https://www.youtube.com/watch?v=8SQV-B83tPU) had them in front of all its transcribed text, and the only reason I'm looking elsewhere is because using Kevin Stratvert's script now turns up the same error every time I try to run it.
